terraform/ec2/win/vpc.tf

```diff
@@ -1,17 +1,38 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT
 
-data "aws_vpc" "default" {
-  default = true
+data "aws_vpc" "vpc" {
```

Comment:
Will create a PR for consolidate the sharing components between EC2 OS https://github.com/aws/amazon-cloudwatch-agent-test/pull/167